src/bgw/job_stat.c

```diff
@@ -123,8 +123,8 @@ bgw_job_stat_tuple_mark_start(TupleInfo *ti, void *const data)
 	catalog_update(ti->scanrel, tuple);
 	heap_freetuple(tuple);
 
-	/* scans with catalog_update must return false */
-	return false;
+	/* scans with catalog_update must return SCAN_DONE */
```

Comment:
Here I think an accurate comment would be helpful because you can't tell from the function name that it's being used as a `tuple_found`.

Reply:
You can tell from the signature, i.e., the return result and parameters, I think.